spec/requests/admin/base_items_requests_spec.rb

```diff
@@ -1,15 +1,25 @@
 RSpec.describe "Admin::BaseItems", type: :request do
   let(:organization) { create(:organization) }
   let(:user) { create(:user, organization: organization) }
+  let(:super_admin) { create(:super_admin, organization: organization) }
   let(:organization_admin) { create(:organization_admin, organization: organization) }
 
-  # TODO: should this be testing something?
   context "while signed in as a super admin" do
     before do
-      sign_in(@super_admin)
+      sign_in(super_admin)
+    end
+
+    it "doesn't let you delete the Kit base item" do
+      kit_base_item = KitCreateService.find_or_create_kit_base_item!
+      delete admin_base_item_path(id: kit_base_item.id)
+      expect(flash[:alert]).to include("You cannot delete the Kits base item")
+      expect(response).to be_redirect
+      expect(BaseItem.exists?(kit_base_item.id)).to be true
     end
   end
 
+  # TODO aren't organization_admins not allowed to view base items?
+  # also, some of these tests are sending organization.id instead of BaseItem.id as args
```

Comment:
Pretty sure they can't. @cielf ?

Reply:
Org admins can't see the base items pages.   

They see a list of base items that they pick from when setting up items,  and there are places they can filter by base item, but they can't view or edit the base item information.

Reply:
fixed specs to clarify that org admins can't access or edit base items in commit [44c42be](https://github.com/rubyforgood/human-essentials/pull/4665/commits/44c42bedf97abdf9f4b8f2206f017c2a729a855f)